apps/topicmaps/bplan-auskunft/src/app/components/Map.tsx

```diff
@@ -15,7 +15,7 @@ import { proj4crs25832def } from 'react-cismap/constants/gis';
 import { getGazData } from '../../utils/gazData';
 import GazetteerSearchControl from 'react-cismap/GazetteerSearchControl';
 import { faSearch } from '@fortawesome/free-solid-svg-icons';
-import { Modal } from '@carma-collab/wuppertal/bplan-auskunft';
+// import { Modal } from '@carma-collab/wuppertal/bplan-auskunft'; TODO FIX IMPORT from commons
```

Comment:
@d4v3000 are there some collab/commons for this?
cant find the imports. needs to be added back in ASAP

same as with InfoBoxText